.travis.yml

```diff
@@ -7,33 +7,41 @@ cache:
 
 env:
   global:
-    - RUN_PHPUNIT=1
     - RUN_PHPCS=0
 
 matrix:
   include:
-    - php: 7.0
-      env: RUN_PHPUNIT=0
-    - php: 7.0
-      env: RUN_PHPUNIT=0 COMPOSER_FLAGS="--prefer-stable --prefer-lowest"
-    - php: 7.1
-      env: RUN_PHPUNIT=0
-    - php: 7.2
-    - php: 7.3
-    - php: 7.4
-      env: RUN_PHPCS=1
+    - php: '7.2'
+      env: LARAVEL='5.5.*'
+    - php: '7.2'
+      env: LARAVEL='5.8.*'
+    - php: '7.2'
+      env: LARAVEL='^6.0'
+    - php: '7.3'
+      env: LARAVEL='5.5.*'
+    - php: '7.3'
+      env: LARAVEL='5.8.*'
+    - php: '7.3'
+      env: LARAVEL='^6.0'
+    - php: '7.4'
+      env: LARAVEL='5.5.*'
+    - php: '7.4'
+      env: LARAVEL='5.8.*'
+    - php: '7.4'
+      env: LARAVEL='^6.0' RUN_PHPCS=1
 
 before_script:
   - phpenv config-rm xdebug.ini || true
 
 install:
-  - |
-    if [[ $RUN_PHPUNIT = 0 ]]; then
-      # We assume the older PHP/Laravel versions and thus restore a working dependency system for them
-      composer remove --dev orchestra/testbench --no-interaction --no-update
-    fi
-  - travis_wait 20 travis_retry composer update ${COMPOSER_FLAGS} --no-interaction --prefer-dist
```

Comment:
`travis_wait` also not necessary anymore, the resolving of the version constraint without `--prefer-lowest` does not require it anymore